<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>full_information</class>
 <widget class="QWidget" name="full_information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(145, 206, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_persian">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>فارسی</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_english">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>انگلیسی</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_home">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>خانه</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_dark_sun">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" rowspan="7">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_sing_company">
            <property name="text">
             <string>اگر شرکت هستید از این جا ثبت نام کنید</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_photo">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>250</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_select_photo">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_ok">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>تایید </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_name">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>نام خود را وارد کنید </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_name">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_last_name">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>نام خانودادگی خود را وارد کنید</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_last_name">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="pushButton_phoneLineEdit">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>شماره ی همراه خود را وارد کنید </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="phoneLineEdit">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="pushButton_emailLineEdit">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>ایمیل خود را وارد کنید</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="emailLineEdit">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1" rowspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_skill">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>مهارت خود را انتخاب کنید </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_skills">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>100</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>c</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>c++</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Qt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PHP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>html</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
